linux/keyman-config/keyman_config/uninstall_kmp.py

```diff
@@ -37,31 +37,32 @@ def uninstall_kmp_shared(packageID):
     """
     kbdir = get_keyboard_dir(InstallLocation.Shared, packageID)
     if not os.path.isdir(kbdir):
-        logging.error("Keyboard directory for %s does not exist. Aborting", packageID)
-        exit(3)
+        msg = _("Keyboard directory for %s does not exist." % packageID)
+        logging.error(msg)
+        return msg
```

Comment:
Did you also try to run this with the command line tool `km-package-uninstall` and checked/tested the other places where `uninstall_kmp` gets called? We want to make sure that it still works if we don't `exit` here.

Reply:
I just tried using `km-package-uninstall` and tested the other places where `uninstall_kmp` gets called and everything works fine for me.